docs/DSA-Problem-Solution/Delete_occurences_of_key.md

```diff
@@ -0,0 +1,263 @@
+---
+id: delete-all-occurences-of-key
+title: "Deleting All Occurrences of a Key in a Doubly Linked List"
+sidebar_label: "Deleting All Occurrences of a Key in a Doubly Linked List
+sidebar_position: 12
+description: "This document provides a detailed explanation and implementation for deleting all occurrences of a key in a doubly linked list , including step-by-step instructions and example code."
+tags: [Linked list,Deletion]
+---
```

Comment:
can not read a block mapping entry; a multiline key may not be an implicit key.
like:
`name: null,`